packages/location-field/src/index.js

```diff
@@ -326,11 +327,19 @@ class LocationField extends Component {
       operatorIconMap
     } = this.props;
     const { activeIndex } = this.state;
+
+    // generate the friendly labels for this feature
+    const { main, secondary } = generateLabel(feature.properties);
+
     // Create the selection handler
     const locationSelected = () => {
       getGeocoder(geocoderConfig)
         .getLocationFromGeocodedFeature(feature)
         .then(geocodedLocation => {
+          // add the friendly location labels for use later on
+          geocodedLocation.main = main;
+          geocodedLocation.secondary = secondary;
+          geocodedLocation.name = getCombinedLabel(feature.properties);
```

Comment:
This is needed to avoid a crash!

```suggestion
          geocodedLocation.name = getCombinedLabel(feature.properties) || '';
```